src/controls/listView/ListView.tsx

```diff
@@ -572,17 +572,40 @@ export class ListView extends React.Component<IListViewProps, IListViewState> {
       return null;
     }
 
+    const headerProps = { 
+      ...props,
+      className: this.props.headerClassName,
+      styles: {
+        root: { 
+          lineHeight: "normal",
+          minHeight: '17px',
+          '.ms-DetailsHeader-cell': {
```

Comment:
Referencing the class name is likely to fail in the future as we have no control over style updates from external components. Can you think of a more reliable approach?